### PR TITLE
Forms Automated Test - LPS-131222 | master

### PR DESCRIPTION
**Pull request by LucasAcacio**

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase [hunk 1]
```diff
@@ -60,12 +60,48 @@ definition {
...
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "1",
+			fieldType = "Paragraph");
+
+		var fieldParagraphField = FormFields.getFieldName();
```
Comment (CalumR23) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase:79]: You are declaring a var here but it is not being referenced elsewhere. Is it possible to take this line out and have the test still pass?

Comment (LucasAcacio) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase]: Yes, I pushed the test with this change

**Comment by CalumR23** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase [hunk 2]
```diff
@@ -60,12 +60,48 @@ definition {
 	}
 
 	@description = "Verify that a Paragraph Field can be edited"
-	@ignore = "true"
 	@priority = "5"
 	test CanBeEdited {
 		property portal.acceptance = "false";
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase:65]: Can you please add `property forms.functional = "true";` here following alphabetic order?

I mean, something like:
```
property forms.functional = "true";
property portal.acceptance = "false";
```

Thanks.

Comment (LucasAcacio) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase:66]: Changes done.

**Comment by CalumR23** [comment]
ci:test:relevant

**Comment by brittneyq** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase [hunk 3]
```diff
@@ -60,12 +60,47 @@ definition {
...
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Paragraph");
+
+		Form.editFieldLabelMetal(fieldValue = "Paragraph");
```
Comment (brittneyq) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase:80]: Hi @LucasAcacio, is line 80 necessary for the test to pass?

**Comment by brittneyq** [comment]
ci:test:sf

**Comment by brittneyq** [comment]
@natocesarrego , @diogocordeiro 

this should be ready for review!

**Comment by diogocordeiro** [comment]
ci:forward

**Pull request closed without merging**